nautobot/extras/templates/extras/job.html

```diff
@@ -19,7 +19,7 @@ <h1>{{ job }}</h1>
     {% if job.read_only %}
         <label class="label label-default">Read-only</label>
     {% endif %}
-    <p>{{ job.description }}</p>
+    <p>{{ job.description | render_markdown | placeholder }}</p>
```

Comment:
A similar `render_markdown` addition is probably needed in `nautobot/extras/templates/extras/jobresult.html`, I think.